products/cos.md

```diff
@@ -0,0 +1,115 @@
+  - releaseCycle: "77"
+    releaseDate: 2019-09-27
+    latest: "cos-77-12371-1109-0"
+    latestReleaseDate: 2021-01-11
+    lts: false
+    eol: 2021-04-01
+    support: 2021-04-01
+  - releaseCycle: "73"
+    releaseDate: 2019-03-25
+    latest: "cos-73-11647-656-0"
+    latestReleaseDate: 2020-09-05
+    lts: false
+    eol: 2020-06-01
+    support: 2020-06-01
+  - releaseCycle: "69"
+    releaseDate: 2018-11-18
+    latest: "cos-69-10895-385-0"
+    latestReleaseDate: 2019-10-08
+    lts: false
+    eol: 2019-12-01
+    support: 2019-12-01
+
+---
+
+> [Google Container-Optimized OS (COS)](https://cloud.google.com/container-optimized-os/docs/concepts/features-and-benefits) is an operating system image for [GCP Compute Engine VMs](https://cloud.google.com/compute) which is optimized for running Docker containers. It is maintained by Google and is based on the open source [Chromium OS](https://www.chromium.org/chromium-os) project.
+
+## Support policy
+
+Container-Optimized OS is tested and qualified for running various container workloads on Google Cloud. It is actively patched with security updates (CVEs) and bug-fixes to address issues that impact Google Cloud users. Users get access to these fixes via image releases in the `cos-cloud` project.
+
+Container-Optimized OS image support is available for Long Term Supported (LTS) family of images in the `cos-cloud` project. A description of available image families is available at the [Versioning Scheme](https://cloud.google.com/container-optimized-os/docs/concepts/versioning#image_families) doc. The following support commitment applies only to the LTS family of images.
+
+## Support duration
+
+Approximately twice each year, the Container-Optimized OS team releases a Long Term Supported (LTS) milestone on its `cos-[MILESTONE]-lts` and `cos-arm64-[MILESTONE]-lts` image families. Once a milestone is introduced on the LTS families, it is supported for **2 years**.
+
+During this support window:
+
+* The milestone is actively patched for any critical bugs.
+* The milestone is actively scanned for security vulnerabilities, and security fixes are applied on regular basis.
+* New releases containing high priority bug and security fixes are released on-demand.
+* New releases containing medium and low priority bug and security fixes are released every 3 months.
+* To maintain stability, no new breaking features are introduced.
+* All changes in the image are documented on a dedicated [release notes](https://cloud.google.com/container-optimized-os/docs/release-notes#lts_image_families) page.
+
+## Deprecation
+
+At the end of a milestone's support window, the corresponding `cos-[MILESTONE]-lts` and `cos-arm64-[MILESTONE]-lts` families are deprecated. Specifically, the [DEPRECATED flag](https://cloud.google.com/compute/docs/reference/latest/images/deprecate) is set on images in that milestone and those images stop appearing in the active list of images in the `cos-cloud` project.
```

Comment:
could you change this link to https://cloud.google.com/compute/docs/reference/rest/v1/images/deprecate
to overcome redirecting problem

Reply:
Done